### PR TITLE
Update Logging Doc

### PR DESCRIPTION
**Pull request by saturnism**
Removed optional projectId, and also fixed the path to the configuration file.

### CODE REVIEW

File: spring-cloud-gcp-docs/src/main/asciidoc/logging.adoc [hunk 1]
```diff
@@ -99,8 +99,7 @@ Your Logback configuration may then look something like this:
 [source, xml]
 ----
 <configuration>
-  <property name="projectId" value="test-project"/>
-  <include resource="org/springframework/cloud/gcp/logging/logback-json-appender.xml" />
+  <include resource="org/springframework/cloud/gcp/autoconfigure/logging/logback-appender.xml" />
```
Comment (joaoandremartins) [inline spring-cloud-gcp-docs/src/main/asciidoc/logging.adoc]: @meltsufin you added this bit, wdyt?

Comment (meltsufin) [inline spring-cloud-gcp-docs/src/main/asciidoc/logging.adoc]: It should be `..logback-json-appender.xml` not `...logback-appender.xml`.

**Comment by meltsufin** [comment]
@saturnism Ping

File: spring-cloud-gcp-docs/src/main/asciidoc/logging.adoc (hunk 1 above)
Comment (ChengyuanZhao) [reply spring-cloud-gcp-docs/src/main/asciidoc/logging.adoc]: @meltsufin PTAL

**Pull request merged**